tests/snappi_tests/intf_utils/intf_accept_metrics.py

```diff
@@ -0,0 +1,109 @@
+# This file defines the interfaces that snappi tests accept external metrics.
+
+#  Metrics data are organized into the hierarchies below
+#  ResourceMetrics
+#    ├── ResourceID
+#    └── ScopeMetrics
```

Comment:
I don't think we need the level of ScopeMetrics.

Reply:
My thought is
Resource level: all metrics from one test run
Scope level: all metrics belonging to one device
Metric level: all metrics belonging to one category
I might be wrong. Let's discuss this topic tomorrow.